expvar/expvar.go

```diff
@@ -46,6 +46,26 @@ func (v *Int) IntValue() int64 {
 	return atomic.LoadInt64(&v.i)
 }
 
+// IntGauge is a 64-bit integer variable that satisfies the expvar.Var interface.
+type IntFuncGauge struct {
+	ValueF func() int
```

Comment:
Should this be `func() int64` so that the function itself can handle the casting?